src/entries/popup/pages/home/NFTs/NFTCollectionSection.tsx

```diff
@@ -49,16 +54,57 @@ export function NFTCollectionSection({
   const { isWatchingWallet } = useWallets();
   const sections = useNftsStore.use.sections();
   const toggleGallerySectionOpen = useNftsStore.use.toggleGallerySectionOpen();
+  const hidden = useNftsStore.use.hidden();
+  const hiddenNftsForAddress = hidden[address] || {};
   const sectionsForAddress = sections[address] || {};
-  const collectionId = section?.collection?.collection_id;
-  const collectionVisible = collectionId && sectionsForAddress[collectionId];
+  const collectionId = collection?.collection_id;
+  const totalCopiesOwned = collection?.distinct_nfts_owned;
+  const isHiddenSection =
+    collection.collection_details.description === '_hidden';
```

Comment:
should we have this `_hidden` in a const or something? feels random but idk, i feel like if i have to touch this code in the future i won't know what's that

Reply:
I can make it a const. I just needed a field on the section to indicate it was the hidden one.